XenAdmin/Dialogs/VMProtectionRecovery/PolicyHistory.cs

```diff
@@ -169,7 +169,13 @@ private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
             if (_policy != null)
             {
                 if (comboBox1.SelectedIndex == 0)
-                    RefreshGrid(_policy.PolicyAlerts);
+		{

```

Comment:
Currently were not respecting the filter hence Policy alerts would have most recent 10 alerts but now we have modified the code to respect the filter so there is a possibility that Policy Alerts would not hold 10 alerts but also 24 hrs and 7 days alerts.